src/index.html

```diff
@@ -1,13 +1,950 @@
+                  Capacity: 370mAh <br>
+                  Battery life: 10h of continuous use 150 hours on stand by <br>
+                  Charging: Fast charging - 1 hour <br>
+                  Charging voltage and current: 5V = 0.5A <br>
+                </p>
+              </div>
+            </div>
+
+            <div class="tech__lines tech__lines--3">
+              <div class="tech__line tech__line--3"></div>
+              <div class="tech__info tech__info--3">
+                <p class="tech__info-title">
+                  CONNECTION
+                </p>
+
+                <p class="tech__text">
+                  Wireless: Bluetooth 4.2 <br>
+                  Signal range: 5m <br>
+                  Receiver: USB 2.0 and above <br>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="kat" id="kat">
+      <div class="container">
+        <h2 class="kat__title">
+          WHY
+          <span class="kat__title--blue">
+            KAT LOCO?
+          </span>
+        </h2>
+
+        <div class="kat__wrapper">
+          <div class="kat__card">
+            <div class="kat__logo kat__logo--1"></div>
+
+            <div class="kat__subtitle">
+              UNIVERSALLY COMPATIBLE
+            </div>
+
+            <div class="kat__text">
+              KAT Loco offers universal compatibility across all major VR headsets and platforms...
+            </div>
+          </div>
+
+          <div class="kat__card">
+            <div class="kat__logo kat__logo--2"></div>
+
+            <div class="kat__subtitle">
+              VR/PC CONTROL PANEL
+            </div>
+
+            <div class="kat__text">
+              Our Multifunctional Software allows for quick access to KAT Loco’s control panel both from...
+            </div>
+          </div>
+
+          <div class="kat__card">
+            <div class="kat__logo kat__logo--3"></div>
+
+            <div class="kat__subtitle">
+              WIRELESS SENSORS
+            </div>
+
+            <div class="kat__text">
+              What makes it even more advanced, KAT Loco is entirely wireless, and comes with a...
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="touch" id="touch">
+      <div class="container">
+        <div class="touch__wrapper">
+          <div class="touch__info">
+            <div class="touch__titles">
+              <p class="touch__subtitle">Have any questions?</p>
+
+              <h3 class="touch__title">
+                GET IN
+                <span class="touch__title--blue">TOUCH</span>
+              </h3>
+
+              <p class="touch__feedback">
+                Our manager will reply you within 15 minutes
+              </p>
+
+              <div class="touch__link-wrapper">
+                <a href="tel:+86-0571-86105373" class="touch__link">
```

Comment:
Need to fix format phone
```suggestion
                <a href="tel:+86057186105373" class="touch__link">
```